package.json

```diff
@@ -1,18 +1,26 @@
 {
   "name": "loopback-workspace",
   "version": "4.0.0-alpha.1",
-  "main": "index.js",
+  "main": "server/server.js",
+  "engines": {
+    "node": ">=4"
+  },
   "scripts": {
-    "test": "node .",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "test": "eslint .",
```

Comment:
Why run the linter twice via npm test? Should be `cucumber` or something. You basically run `eslint .` and then run `npm run lint` again in post test after.